README.md

```diff
@@ -1 +1,19 @@
 ### Moya Contributors
+
+##### Background
+
+Moya started out as a project in Artsy under the ownership of [Ash Furrow](/ashfurrow) and [Orta Therox](/orta). Over time Moya became too big for us to maintain within the Artsy product cycle.
```

Comment:
I get that you're referencing that it's an organization now, but it sounds kind of like Moya is this monster that grew outside of our control.

![](http://imagesmtv-a.akamaihd.net/uri/mgid:file:http:shared:mtv.com/news/wp-content/uploads/2014/12/tumblr_mnb87rsKgB1s5or7ko3_500-1418053915.gif)

Maybe instead, it could be:

> Over time, developers from the community began using Moya and it is now a community-driven project.